### DISCUSSION TITLE
Can I switch off the 're-run' functionality

### DISCUSSION THREAD

**Discussion by AndreasJHeix** [concourse/concourse#6562] [Help & Support] [answered]
Hi,
https://concourse-ci.org/jobs.html#fly-rerun-build  is relatively prominent in the UI and I am concerned that somebody hits it by accident when browsing through old job logs. That could lead to downgrade of a service being deployed with this job.

=> Is there a way we can disable this link from the UI? Or disable the complete rerun-functionality if needed?

Thanks for your help!
Andreas

**Comment by vito** [comment, marked as answer]
This does not exist, but it would be easy to implement just like [`disable_manual_trigger`](https://concourse-ci.org/jobs.html#schema.job.disable_manual_trigger) if anyone is interested in submitting a PR.
